docs/rum_collection/data_collected.md

```diff
@@ -0,0 +1,237 @@
+---
+title: RUM iOS Data Collected
+kind: documentation
+further_reading:
+  - link: "https://github.com/DataDog/dd-sdk-ios"
+    tag: "Github"
+    text: "dd-sdk-ios Source code"
+  - link: "/real_user_monitoring"
+    tag: "Homepage"
+    text: "Explore Datadog RUM"
+---
+
```

Comment:
Take a look at how I restructured the Browser Data Collected page, and see if you want to do the same thing here: https://docs.datadoghq.com/real_user_monitoring/browser/data_collected/ 

The page talks about Default attributes first (because it applies to all RUM data), and then it talks about Event-specific ones second (because people will pick and choose what type they are dealing with). And instead of presenting the Event-specific data on tabs, we flattened it so that Cmd+F works for people looking for "all the data."

Reply:
I see 👍 , this is a very good idea. Let me update it 🙂

Reply:
I've just committed this change.